src/graph/graph_delete_nodes.c

```diff
@@ -59,7 +59,7 @@ void Graph_DeleteNodes
 	//--------------------------------------------------------------------------
 
 	GrB_Index j;            // iterated entry col idx
-	GrB_Scalar s;           // empty scalar
+	GrB_Matrix s;           // empty scalar
```

Comment:
We can go back to a scalar, if we choose not to then please update the comment to: `// empty matrix`